doc/controls/AutoLayoutControl.md

```diff
@@ -0,0 +1,45 @@
+> [!TIP] 
+> This guide covers details for `AutoLayoutControl` specifically. If you are just getting started with the Uno Toolkit UI Library, please see our [general getting started](../getting-started.md) page to make sure you have the correct setup in place.
+
+## Summary
+`AutoLayoutControl` arranges child elements into a single line that can be oriented horizontally or vertically. It is the reflection in XAML of the [Autolayout frame in Figma](https://www.figma.com/widget-docs/api/component-AutoLayout).
+
+### C#
+```csharp
+public partial class AutoLayout : Control
+```
+
+### XAML
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:AutoLayout>
+    contents
+</utu:AutoLayout>
+```
+
+### Inheritance 
+Object &#8594; DependencyObject &#8594; UIElement &#8594; FrameworkElement &#8594; Control
+
+### Constructors
+| Constructor     | Description                                              |
+|-----------------|----------------------------------------------------------|
+| AutoLayout() | Initializes a new instance of the `AutoLayout` class. |
+
+### Properties
+Property|Type|Description
+-|-|-
+Orientation | Orientation | Defines constants that specify the different orientations that a control or layout can have. 
+Spacing | double | Gets or sets the uniform distance (in pixels) between grid. Each grid of the AutoLayout will have the spacing set by this value depending the AautoLayout orientation
+Justify | AutoLayoutJustify | Ether Stack, space between children behave like a StackPanel, or SpaceBetween, between each children a equal space is added assuring that all the AutoLayout space is occupy. Note : if a children have his PrimaryAlignment set to Stretch, it will default to Stack.
```

Comment:
```suggestion
Justify | AutoLayoutJustify | Ether Stack, space between children behave like a `StackPanel`, or `SpaceBetween`, between each children a equal space is added assuring that all the `AutoLayout` space is occupy. Note : if a child has his `PrimaryAlignment` set to Stretch, it will default to Stack.
```